keyboards/dz60/keymaps/MintCollie/readme.md

```diff
@@ -0,0 +1,48 @@
+## MintCollies DZ60
+Custom layout for the DZ60.
+
+![Photo of the keyboard after I built it](https://i.imgur.com/x0ooI8q.jpeg)
```

Comment:
```suggestion
![Photo of the keyboard after I built it](https://i.imgur.com/x0ooI8qh.jpeg)
```

Reply:
What does adding the ```h``` do? I'm very curious!

Reply:
The H uses a smaller resolution version of the image, one that is more appropriate for the github markdown rendering.    It's a neat little hack, for imgur.